packages/shared/src/components/post/BasePostContent.tsx

```diff
@@ -27,7 +27,7 @@ export function BasePostContent({
   customNavigation,
 }: BasePostContentProps): ReactElement {
   const { id } = post ?? {};
-  const { onCloseShare, sharePost, onSharePost } = engagementProps;
+  const { onCloseShare, sharePost, onCopyPostLink } = engagementProps;
```

Comment:
Have made this renaming, will continue with all renaming in the next PR that was a follow up for cleaning up after coly link experiment. https://dailydotdev.atlassian.net/browse/MI-233